src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_moved_to_error_queue.cs

```diff
@@ -0,0 +1,168 @@
+            .Should(c => Assert.IsFalse(c.OutgoingMessageSent, "Outgoing messages should not be sent"))
+            .Run();
+        }
+
+        [Test]
+        public async Task May_send_outgoing_messages_without_transport_transactions()
+        {
+            await Scenario.Define<Context>(c =>
+            {
+                c.Id = Guid.NewGuid();
+                c.TransactionMode = TransportTransactionMode.None;
+            })
+            .WithEndpoint<Endpoint>(b => b.DoNotFailOnErrorMessages()
+                .When((session, context) => session.SendLocal(new InitiatingMessage
+                {
+                    Id = context.Id
+                }))
+            )
+            .WithEndpoint<ErrorSpy>()
+            .Done(c => c.MessageMovedToErrorQueue)
+            .Repeat(r => r.For<AllDtcTransports>())
+            .Run();
+        }
+
+        const string ErrorSpyQueueName = "error_spy_queue";
+
+        class Context : ScenarioContext
+        {
+            public Guid Id { get; set; }
+            public bool MessageMovedToErrorQueue { get; set; }
+            public bool OutgoingMessageSent { get; set; }
+            public TransportTransactionMode TransactionMode { get; set; }
+        }
+
+        class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint()
+            {
+                EndpointSetup<DefaultServer>((config, context) =>
+                {
+                    var testContext = context.ScenarioContext as Context;
+
+                    config.UseTransport(context.GetTransportType())
+                        .Transactions(testContext.TransactionMode);
+                    config.DisableFeature<FirstLevelRetries>();
+                    config.DisableFeature<SecondLevelRetries>();
+                    config.Pipeline.Register(new RegisterThrowingBehavior());
+                    config.SendFailedMessagesTo(ErrorSpyQueueName);
+                });
+            }
+
+            class InitiatingHandler : IHandleMessages<InitiatingMessage>
+            {
+                public Context TestContext { get; set; }
+
+                public async Task Handle(InitiatingMessage initiatingMessage, IMessageHandlerContext context)
+                {
+                    if (initiatingMessage.Id == TestContext.Id)
+                    {
+                        await context.Send(ErrorSpyQueueName, new OutgoingMessage
+                        {
+                            Id = initiatingMessage.Id
+                        });
+                    }
+                }
+            }
+        }
+
+        class ErrorSpy : EndpointConfigurationBuilder
+        {
+            public ErrorSpy()
+            {
+                EndpointSetup<DefaultServer>(config => config.LimitMessageProcessingConcurrencyTo(1))
+                    .CustomEndpointName(ErrorSpyQueueName);
+            }
+
+            class ErrorHandler : IHandleMessages<InitiatingMessage>
```

Comment:
In a previous test `ErrorHandler` implemented both `IHandleMessages<InitiatingMessage>` and `IHandleMessages<OutgoingMessage` I think the tests should be aligned. I'm in favour of having to classes but that is just my 2 cents.